src/Compilers/CSharp/Portable/Symbols/TypeSymbol.cs

```diff
@@ -861,6 +861,15 @@ private static Symbol ComputeImplementationForInterfaceMember(Symbol interfaceMe
                 CheckForImplementationOfCorrespondingPropertyOrEvent((MethodSymbol)interfaceMember, implementingType, implementingTypeIsFromSomeCompilation, ref implicitImpl);
             }
 
+            if ((object)implicitImpl == null && (object)closestMismatch == null)
+            {
+                // Check for default interface implementations
+                if (!interfaceMember.IsAbstract)
+                {
+                    implicitImpl = interfaceMember;
+                }
+            }
+
```

Comment:
There is absolutely no risk to forget adjusting this place once the support is added, unless there would be no testing, which won't happen.